src/fpm_compiler.f90

```diff
@@ -754,13 +793,15 @@ end function enumerate_libraries
 
 
 !> Create new compiler instance
-subroutine new_compiler(self, fc, cc, echo, verbose)
+subroutine new_compiler(self, fc, cc, cppc, echo, verbose)
     !> New instance of the compiler
     type(compiler_t), intent(out) :: self
     !> Fortran compiler name or path
     character(len=*), intent(in) :: fc
     !> C compiler name or path
     character(len=*), intent(in) :: cc
+    !> C++ Compiler name or path
+    character(len=*), intent(in) :: cppc
```

Comment:
```suggestion
    character(len=*), intent(in) :: cxx
```

I think the convention for this variable name is `CXX` with other build systems.